include/aspect/material_model/interface.h

```diff
@@ -217,6 +217,16 @@ namespace aspect
                           const bool use_strain_rates = true);
 
       /**
+       * Copy Constructor.
+       */
+      MaterialModelInputs(const MaterialModelInputs<dim> &material);
+
+      /**
+       * Assignment operator.
+       */
+      MaterialModelInputs &operator=(const MaterialModelInputs &material);
```

Comment:
Ah, wires crossed. The `=delete` approach doesn't work because that's C++11. We need to come up with something else.